Gemfile.lock

```diff
@@ -251,15 +251,25 @@ GEM
       i18n (>= 1.6, < 2)
     fakeredis (0.8.0)
       redis (~> 4.1)
-    faraday (1.4.1)
+    faraday (1.7.2)
```

Comment:
there's a dependency of oauth2 that prevents _us_ from accidentally jumping to the faraday 2.0 branch (their current main branch) - I _think_ these updates are safe